db-server/test/backend/db_tests.js

```diff
@@ -748,7 +748,7 @@ module.exports = function (config, DB) {
         .then(function (account) {
           assert(account.emailVerified, 'account should now be emailVerified (truthy)')
           assert.equal(account.emailVerified, 1, 'account should now be emailVerified (1)')
-          assert.equal(account.profileChangedAt > account.createdAt, true, 'profileChangedAt updated')
+          assert.equal(account.profileChangedAt >= account.createdAt, true, 'profileChangedAt updated')
```

Comment:
As with the oauth-server PR, I wonder if we should leave a note on these tests to explain why they look funny.  Without context, this just looks like we typoed `>=` instead of `>` when `>` is the "obviously correct" behaviour.